Don't try to build with -DG_DISABLE_DEPRECATED - glib has moved on

source: https://src.fedoraproject.org/cgit/rpms/gamin.git/plain/gamin-manape.patch

--- a/server/Makefile.am.manape	2008-11-12 15:45:28.000000000 -0500
+++ b/server/Makefile.am	2011-07-20 21:06:37.465307638 -0400
@@ -6,8 +6,7 @@ INCLUDES =						\
 	-I$(top_srcdir)/lib				\
 	-I$(top_srcdir)/libgamin			\
 	$(DAEMON_CFLAGS)				\
-	-DBINDIR=\""$(libexecdir)"\"			\
-	-DG_DISABLE_DEPRECATED				
+	-DBINDIR=\""$(libexecdir)"\"
 
 if GAMIN_DEBUG
 INCLUDES += -DGAM_DEBUG_ENABLED
